src/open_inwoner/scss/components/Product/product-location.scss

```diff
@@ -0,0 +1,14 @@
+.location {
+  padding: 0 0 var(--spacing-giant) 0;
+
+  &__details {
+    display: flex;
+    gap: var(--spacing-extra-large);
+    margin-top: var(--spacing-small);
+  }
+
+  &__address p:nth-child(2),
+  &__contact p:nth-child(2) {
```

Comment:
Why `nth-child(2)`  here? Feels like a magic number

Reply:
I think I made this more generic for purposes like you mentioned (multiple <p>'s)